src/components/tracked-controls.js

```diff
@@ -145,12 +145,12 @@ module.exports.Component = registerComponent('tracked-controls', {
     // Compose pose from Gamepad.
     pose = controller.pose;
     // If no orientation, use camera.
-    if (pose.orientation) {
+    if (pose && pose.orientation) {
```

Comment:
it would be possible to have a controller with position but not orientation, so guard for absence of either